contracts/tasks/aero.js

```diff
@@ -0,0 +1,162 @@
+const { formatUnits } = require("ethers").utils;
+const { BigNumber } = require("ethers");
+const { getBlock } = require("./block");
+const { resolveAsset, resolveContract } = require("../utils/resolvers");
+const { getSigner } = require("../utils/signers");
+const { base } = require("../utils/addresses");
+
+const snapAero = async ({ block }) => {
+  const signer = await getSigner();
+  const blockTag = await getBlock(block);
+
+  const weth = await resolveAsset("WETH");
+  const vault = await resolveContract("OETHBaseVaultProxy", "IVault");
+  const oeth = await resolveContract("OETHBaseProxy", "OETHBase");
+  const pool = await resolveContract(base.aerodromeOETHbWETHClPool, "ICLPool");
+  const aeroStrat = await resolveContract(
+    `AerodromeAMOStrategyProxy`,
+    "AerodromeAMOStrategy"
+  );
+  const sugarHelper = await resolveContract(base.sugarHelper, "ISugarHelper");
+
+  const Q96 = BigNumber.from(2).pow(96);
+  const sqrtRatioX96TickLower = BigNumber.from("79224201403219477170569942574"); // -1 tick
```

Comment:
These could be read from the strategy contract. Would make it easier to use the same script for different strategies:
https://github.com/OriginProtocol/origin-dollar/blob/master/contracts/contracts/strategies/aerodrome/AerodromeAMOStrategy.sol#L87

Reply:
Good idea. I'll add